packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -242,7 +242,7 @@ function Inputs({
       options={options}
       placeholder={placeholder ? { id: placeholder, defaultMessage: placeholder } : null}
       required={fieldSchema.required || false}
-      step={step}
+      step={metadataStep || step}
```

Comment:
If a user entered `00:04` as a time, but then change the field interval changes to `10`, the field would show `00:00` (instead of the real value), which might be confusing.

What do you think about checking, whether the current value matches the selected interval:

1. if yes: use `metadataStep`
2. if no: use `1`

Reply:
Sounds good to me. When editing a collection/single type, if the current time/datetime field is empty or matches the selected interval then the `metadataStep` will be used, otherwise, it'll use the default `step`.